test/config/credentials_default.json

```diff
@@ -19,8 +20,8 @@
     ],
     "ora": [
       {
-        "username": "vagrant",
-        "password": "vagrant"
+        "username": "onrack",
+        "password": "onrack"
```

Comment:
OK.  These should be vagrant/vagrant then as Stuart said.  What about other "onrack" credentials:

  "node": [
      {
        "username": "root",
        "password": "1234567"
      },
      {
        "username": "onrack",
        "password": "onrack"
      }
    ],